app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -153,14 +153,13 @@ class LuciBuildService {
         'repo_name': slug.name,
         'user_agent': 'flutter-cocoon',
       };
-      if (checkSuiteEvent != null) {
-        final github.CheckRun checkRun =
-            await githubClient.checks.checkRuns.createCheckRun(
-          checkSuiteEvent.repository.slug(),
-          name: builder,
-          headSha: commitSha,
+      if (checkSuiteEvent != null || config.checksSupportedRepo(slug)) {
```

Comment:
`checkSuiteEvent` does not seem to be available any more.

Reply:
It is available is just that we are workaround it because of a failure in the checks api.